packages/code-du-travail-data/dataset/datafiller/fetch-themes.js

```diff
@@ -0,0 +1,89 @@
+const fetch = require("node-fetch");
+const slugify = require("../../slugify");
+
+const { sortByKey, getVariants, sortRowRefs, slimify } = require("./utils");
+
+/*
+ fetch raw datafiller themes data, filter and sort properly
+*/
+
+const DATAFILLER_URL =
+  process.env.DATAFILLER_URL || "https://datafiller.num.social.gouv.fr";
+
+const RECORDS_URL = `${DATAFILLER_URL}/kinto/v1/buckets/datasets/collections/themes/records?_limit=1000`;
```

Comment:
does _limit=1000 needed here if you pass it while fetching ?

Reply:
removed